persistence/sql/identity/persister_identity.go

```diff
@@ -526,6 +605,14 @@ func (p *IdentityPersister) ListIdentities(ctx context.Context, params identity.
 		return nil, err
 	}
 
+	if len(params.Expand) > 0 {
```

Comment:
I did not realize that we now need to do this. This will have a significant performance impact on the identity list endpoint, because we are iteratively stepping over the result set. This can easily expand to 400-500 waterfall-like SQL queries. I almost fear that this is a blocker because people will immediately complain about it.

Reply:
Fixed, I think. The code is a little less nice now, but still worth it IMO.

LMKWYT.